tests/test_integration/test_stencil_object.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# GT4Py - GridTools4Py - GridTools for Python
+#
+# Copyright (c) 2014-2021, ETH Zurich
+# All rights reserved.
+#
+# This file is part the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""Integration tests for StencilObjects."""
+
+from typing import Any, Dict
+
+import pytest
+
+from gt4py import gtscript
+from gt4py import storage as gt_storage
+from gt4py.gtscript import PARALLEL, Field, computation, interval
+
+
+@pytest.fixture
+def backend() -> str:
+    return "gtc:numpy"
+
+
+def test_stencil_object_cache(backend: str):
```

Comment:
I think creating a fixture here is a bit of an overkill, so I suggest to use a parametrize argument:

```suggestion
@pytest.mark.parametrize("backend", ["gtc:numpy"])
def test_stencil_object_cache(backend: str):
```